CRM/Utils/Hooks.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/*
+ +-------------------------------------------------------+
+| 2020 SYSTOPIA                                          |
+| Author: P. Batroff (batroff@systopia.de)               |
+ +-------------------------------------------------------+
+*/
+
+/**
+ * Defines the hooks that allow specialized bounce handling
+ */
+class CRM_Utils_Hooks {
```

Comment:
This should be **CRM_Mailjet**_Utils_Hooks (or CRM_Mailjet_Hooks if you want to cut down on directory depth) to avoid a conflict with core classes.

Reply:
that doesn't work then with the directory structure. The (possibly) conflicting class would be `abstract class CRM_Utils_Hook` in the civicrm sources. I would be ok with renaming the class, but then we would have to move the class to the Mailjet folder, otherwise it wouldn't be found

Reply:
e.g. Move the hooks file to Mailjet/Hooks.php, the we would need to rename it to `CRM_Mailjet_Hooks`

Reply:
Yes exactly, that's what I meant in my first review